src/libcrun/utils.c

```diff
@@ -897,8 +897,23 @@ libcrun_is_apparmor_enabled (libcrun_error_t *err)
   return apparmor_enabled;
 }
 
+static int 
+is_current_process_confined(libcrun_error_t *err)
+{
+  int fd = open("/proc/thread-self/attr/current", O_RDONLY|O_CLOEXEC);
```

Comment:
I did a test on Ubuntu 22.04.3
```
$ cat /proc/thread-self/attr/current
unconfined
$ cat /proc/thread-self/attr/apparmor/current
unconfined
$
```

I saw this

* https://github.com/containers/crun/issues/1350

Maybe the path _/proc/thread-self/attr/apparmor/current_ should be used too somehow?

disclaimer: I don't know anything about apparmor so I'm just guessing.

Reply:
you are right, it is better to use the lsm specific attr.